src/core/types/ui_settings.ts

```diff
@@ -65,6 +66,13 @@ export interface UiSettingsParams<T = unknown> {
   type?: UiSettingsType;
   /** optional deprecation information. Used to generate a deprecation warning. */
   deprecation?: DeprecationSettings;
+  /**
+   * index of the settings within its category (ascending order, smallest will be displayed first).
+   * Used for ordering in the UI.
+   *
+   * @remark settings without order defined will be displayed last and ordered by name
+   */
+  order?: number;
```

Comment:
So, as the first implementation was supposed to be per-space (basically phase 2 without phase 1), I already performed the following changes to `uiSettings` before we changed our mind:

- Add a new `order` field to sort the field in their category
- Add a new `color` type to `UiSettingsType`

As this implementation is no longer based on `uiSettings`, we could revert that. However, given that these will be used in phase 2 anyway, and as I also already modified the `advancedSettings` app to handle them, I feel like we can keep them in this PR.

WDYT?

Reply:
Sounds good